### PR TITLE
new material type-------bean

### PR DESCRIPTION
**Pull request by LiGIAOTPA**
Beans and comestibles made from been products belong to been now, those stuffs are full of calcium , vitamin A, and i also added soy milk in the recipes : )

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Using materials for food causes a set of problems. I described them in #21395 to avoid discussing them here. Still, #21395 needs to be resolved before this can be merged.

File: data/json/items/comestibles.json [hunk 1]
```diff
@@ -390,6 +390,28 @@
...
+        "weight" : 260,
+        "color" : "white",
+        "spoils_in" : 8,
+        "container" : "jug_plastic",
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 30,
+        "healthy" : 1,
+        "nutrition" : 4,
+        "description" : "a substitute for milk, made from beans.",
```
Comment (Coolthulhu) [inline data/json/items/comestibles.json]: Capital letter

File: data/json/items/comestibles.json [hunk 2]
```diff
@@ -390,6 +390,28 @@
...
+        "name_plural": "soymilk",
+        "weight" : 260,
+        "color" : "white",
+        "spoils_in" : 8,
+        "container" : "jug_plastic",
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 30,
+        "healthy" : 1,
+        "nutrition" : 4,
```
Comment (Coolthulhu) [inline data/json/items/comestibles.json]: If you're copying most of the data from cow milk, you should use `"copy-from": "milk"` and only include changes from cowmilk here. Changes like material, description, name etc. - everything that is the same should be left unspecified and handled by "copy-from".

Comment (LiGIAOTPA) [inline data/json/items/comestibles.json]: Thank for your guidance, I am really have no experience in program.

**Comment by kevingranade** [comment]
Agreed with Coolthulu, if we want to continue bulking up nutrition data we need to model it explicitly instead of adding it to materials.

**Pull request closed without merging**